Core/include/Acts/Seeding/SeedFilter.hpp

```diff
@@ -19,6 +19,17 @@
 #include <vector>
 
 namespace Acts {
+struct SeedConfQuantitiesConfig {
```

Comment:
Nitpicking! Sorry. Is this realy a config object?

Reply:
should I rename it to SeedConfQuantitiesStruct or something similar?

Reply:
What about SeedingState ?

Reply:
sounds good to me